g2core/board/ArduinoDue/board_gpio.cpp

```diff
@@ -56,39 +56,39 @@
 
 /**** Setup Actual Objects ****/
 
-gpioDigitalInputPin<IRQPin<Motate::kInput1_PinNumber>>  din1  {DI1_MODE,  1};
-gpioDigitalInputPin<IRQPin<Motate::kInput2_PinNumber>>  din2  {DI2_MODE,  2};
-gpioDigitalInputPin<IRQPin<Motate::kInput3_PinNumber>>  din3  {DI3_MODE,  3};
-gpioDigitalInputPin<IRQPin<Motate::kInput4_PinNumber>>  din4  {DI4_MODE,  4};
-gpioDigitalInputPin<IRQPin<Motate::kInput5_PinNumber>>  din5  {DI5_MODE,  5};
-gpioDigitalInputPin<IRQPin<Motate::kInput6_PinNumber>>  din6  {DI6_MODE,  6};
-gpioDigitalInputPin<IRQPin<Motate::kInput7_PinNumber>>  din7  {DI7_MODE,  7};
-gpioDigitalInputPin<IRQPin<Motate::kInput8_PinNumber>>  din8  {DI8_MODE,  8};
-gpioDigitalInputPin<IRQPin<Motate::kInput9_PinNumber>>  din9  {DI9_MODE,  9};
-// gpioDigitalInputPin<IRQPin<Motate::kInput10_PinNumber>> din10 {DI10_MODE, 10};
-// gpioDigitalInputPin<IRQPin<Motate::kInput11_PinNumber>> din11 {DI11_MODE, 11};
-// gpioDigitalInputPin<IRQPin<Motate::kInput12_PinNumber>> din12 {DI12_MODE, 12};
+gpioDigitalInputPin<IRQPin<Motate::kInput1_PinNumber>>  din1  {DI1_ENABLED,  DI1_POLARITY,  1, DI1_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput2_PinNumber>>  din2  {DI2_ENABLED,  DI2_POLARITY,  2, DI2_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput3_PinNumber>>  din3  {DI3_ENABLED,  DI3_POLARITY,  3, DI3_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput4_PinNumber>>  din4  {DI4_ENABLED,  DI4_POLARITY,  4, DI4_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput5_PinNumber>>  din5  {DI5_ENABLED,  DI5_POLARITY,  5, DI5_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput6_PinNumber>>  din6  {DI6_ENABLED,  DI6_POLARITY,  6, DI6_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput7_PinNumber>>  din7  {DI7_ENABLED,  DI7_POLARITY,  7, DI7_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput8_PinNumber>>  din8  {DI8_ENABLED,  DI8_POLARITY,  8, DI8_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kInput9_PinNumber>>  din9  {DI9_ENABLED,  DI9_POLARITY,  9, DI9_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
+gpioDigitalInputPin<IRQPin<Motate::kSocket2_SPISlaveSelectPinNumber>> din10 {DI10_ENABLED, DI10_POLARITY, 10, DI10_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
```

Comment:
This was a copy-paste of a hack on the G2v9k board to steal an unused slave select for an extra input. Please use `kInput10_PinNumber` instead:
```suggestion
gpioDigitalInputPin<IRQPin<Motate::kInput10_PinNumber >> din10 {DI10_ENABLED, DI10_POLARITY, 10, DI10_EXTERNAL_NUMBER, Motate::kPinInterruptOnChange|Motate::kPinInterruptPriorityHigh};
```